src/service/video-manager-impl.js

```diff
@@ -134,6 +174,48 @@ export class VideoManager {
       this.scrollListenerInstalled_ = true;
     }
   }
+
+  /**
+   * Returns the entry in the video manager corresponding to the video
+   * provided
+   *
+   * @param {!../video-interface.VideoInterface} video
+   * @return {VideoEntry} entry
+   * @private
+   */
+  getEntryForVideo_(video) {
+    for (let i = 0; i < this.entries_.length; i++) {
+      if (this.entries_[i].video === video) {
+        return this.entries_[i];
+      }
+    }
+    dev().assert(false, 'video is not registered to this video manager');
+    return null;
```

Comment:
I think `return null` is necessary because otherwise `check-types` would complain about missing return statement `Missing return statement. Function expected to return VideoEntry$$module$src$service$video_manager_impl.
  getEntryForVideo_(video) {`